### ISSUE TITLE
Binaries for v0.10.0

### ISSUE THREAD

**Issue by duijf** [channable/vaultenv#69]
I couldn't get the new CI upload to work in time for the weekend. There is a binary compiled on Ubuntu 18.04 available on the releases page. I'll look into the rest on Tuesday.

**Comment by lpaulmp** [comment]
Is something that I can help with?
You might have enabled travis-ci on for the repository first https://travis-ci.org/channable/vaultenv

**Comment by duijf** [comment]
> You might have enabled travis-ci on for the repository first travis-ci.org/channable/vaultenv


I enabled it on https://travis-ci.com/channable/vaultenv (`.com` instead of `.org`) last Friday as we already had a corp account with them 

The build works fine, but it seems [to fail during release creation + uploading](https://travis-ci.com/channable/vaultenv/jobs/206363322#L492-L502).

Log excerpts:

```
github-release: user error (Failed to create release! Reason: Right (fromList [("documentation_url",String "https://developer.github.com/v3/repos/releases/#create-a-release"),("message",String "Not Found")]))
github-release: user error (Failed to get upload URL: fromList [("documentation_url",String "https://developer.github.com/v3/repos/releases/#get-a-release-by-tag-name"),("message",String "Not Found")])
```

I suspect it's a permission issue, since GitHub is known to return 404s in case of incorrect tokens: https://developer.github.com/v3/troubleshooting/#why-am-i-getting-a-404-error-on-a-repository-that-exists

So that's something that I'll need to dig into on our side. Some permissions probably need to be bumped somewhere.

**Comment by nabadger** [comment]
@duijf the binary on the release page for `v0.10.0` is actually `v0.9.0` - we spent a while figuring out why the kv version-2 updates were not working, I'm guessing it's this ;)

---

```
wget https://github.com/channable/vaultenv/releases/download/v0.10.0/vaultenv-linux-ubuntu1804  && chmod +x ./vaultenv-linux-ubuntu1804 && ./vaultenv-linux-ubuntu1804 --version
```

**Comment by alexhumphreys** [comment]
Just ran into this issue too, that when you download the [v0.10.0](https://github.com/channable/vaultenv/releases/tag/v0.10.0) you get the `0.9.0`. Any chance of a new release to fix this?

**Comment by tomwassenberg** [comment]
I'm not sure what went wrong here initially, but I just updated the [0.10.0 release](https://github.com/channable/vaultenv/releases/tag/v0.10.0) with a manually built binary + .deb package of the `v0.10.0` tag, built on a Ubuntu 18.04 machine.

Let us know if it works for you now :)

**Comment by alexhumphreys** [comment]
Nice one! I'm on Ubuntu 16.04 and hit this error:

```
$ vaultenv --version
vaultenv: /lib/x86_64-linux-gnu/libm.so.6: version `GLIBC_2.27' not found (required by vaultenv)
```

But I'll be running it in docker anyway, and I grabbed the docker image `ubuntu:18:04` and installed the `.deb` there and it worked so that's me sorted. Cheers!

**Comment by orinokai** [comment]
Is it possible to also get a binary and `.deb` file for v0.10.0 that links against an earlier version of libc? Ideally <=2.24 as I use a number of Docker images that only offer Debian Stretch (e.g. MySQL). I have managed to compile vaultenv manually on Stretch in the meantime, but it would be really useful to get a `.deb` from the release page. Thanks!

**Comment by duijf** [comment]
We've started work on fully static binaries with `musl` in https://github.com/channable/vaultenv/issues/43

This would lift the libc requirements from future releases

**Comment by duijf** [comment]
Vaultenv 0.11 has static binaries available for Linux using Musl https://github.com/channable/vaultenv/releases/tag/v0.11.0

We do not plan to make binaries available for 0.10 as 0.11 is compatible.
